Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -1047,19 +1047,22 @@ final class KSRAnalyticsTests: TestCase {
     ksrAnalytics.trackProjectCardClicked(
       page: .discovery,
       project: .template,
+      typeContext: .project,
```

Comment:
Did you verify on airtable that the type is always .project for this event unless it is clicked after a search query?